lib/elastic_record/index.rb

```diff
@@ -96,6 +96,15 @@ def get(end_path, json = nil)
 
     private
 
+      def add_suffix(name)
+        suffix = ElasticRecord::Config.index_suffix
+        if suffix && !name.end_with?(suffix)
```

Comment:
What is the `end_with?` check for?

Reply:
`add_suffix` is called from `alias_name` and `alias_name=`, so I didn't want to end up with "_test_test", etc, added.